### PR TITLE
Add NMEA GST Message for GPSInformation.Accuracies

### PR DESCRIPTION
**Pull request by jiargei**
## Description

The core feature "GPSInformation" allways shows -1.0m vor Vertical and Horizontal Accuracy... This is handled by the NMEA message GST which was not implemented yet... I added that. 

As "accuracy" definition I used the approach described at [Calculating Vrms and Hrms from nmea data](https://gis.stackexchange.com/questions/217481/calculating-vrms-and-hrms-from-nmea-data/315536?newreg=84d02d6dd2934cd6ac139e9f94818aa3)

Fixes #21933

### CODE REVIEW

**Comment by luipir** [comment]
hi @jiargei thanks for you contribution
I'm not used with this part of code... but did you check if an updated version of the external lib http://nmea.sourceforge.net can solve already the problem?

**Comment by luipir** [comment]
seems not, nmea upstream trunk does not manage GPGST... make sense to contribute upstream to nemea library and then update our repo?
Just my2c

**Comment by jiargei** [comment]
Hej @luipir 
No, i posted that issue there too.... sorry for that proken build.. I thought I don't have to stay to the checklist... proven wrong as so often... 
I'm building it right now... if it works, i send my changes to nmealib too....

**Comment by luipir** [comment]
> if it works, i send my changes to nmealib too

great

**Comment by jiargei** [comment]
Hej @luipir !

It's working...

**Comment by luipir** [comment]
BTW I would wait and align to upstream nmea lib if you are not in a rush

File: external/nmea/parse.c [hunk 1]
```diff
@@ -133,25 +133,28 @@ int nmea_pack_type( const char *buff, int buff_sz )
...
     "GPRMC",
     "GPVTG",
-    "GNRMC",
+    "GNRMC",    
+    "GPGST",
   };
 
   NMEA_ASSERT( buff );
 
   if ( buff_sz < 5 )
```
Comment (jef-n) [inline external/nmea/parse.c]: < 6

Comment (jiargei) [reply external/nmea/parse.c]: fixed that...

**Comment by luipir** [comment]
@jiargei any news from upstream ? can you share the PR link to the upstream library?

**Comment by nyalldawson** [comment]
I've re-tagged this as a bug fix -- it fixes an existing feature which doesn't work

**Comment by jiargei** [comment]
Hej! I appreciate that! Nice work btw!

**Comment by lbartoletti** [comment]
> @jiargei any news from upstream ? can you share the PR link to the upstream library?

For reference https://sourceforge.net/p/nmea/bugs/18/

**Comment by nyalldawson** [comment]
Superseded by #30274

**Comment by jiargei** [comment]
Hej @nyalldawson... If this issue is closed does it mean it wont get implemented?

**Comment by nyalldawson** [comment]
@jiargei  - no, see the note in #30274

**Pull request closed without merging**